repository/a.json

```diff
@@ -1974,6 +1974,18 @@
 				}
 			]
 		},
+		{
+			"name": "AssistantAI",
+			"details": "https://github.com/kanutron/AssistantAI",
+			"author": "kanutron",
```

Comment:
This is simply your username, so you can omit it.

Reply:
OK. Removed the 'author' key.